crates/types/protobuf/restate/common.proto

```diff
@@ -63,6 +64,9 @@ enum TargetName {
   // ReplicatedLoglet
   REPLICATED_LOGLET_APPEND = 40;
   REPLICATED_LOGLET_APPENDED = 41;
+
+  PARTITION_CREATE_SNAPSHOT_REQUEST = 42;
+  PARTITION_CREATE_SNAPSHOT_RESPONSE = 43;
```

Comment:
@AhmedSoliman I see you left some gaps elsewhere - would you like me to start these in a new range, e.g. 50+? I think we'll have more RPCs for managing worker nodes that could be grouped together.

Reply:
Sounds like a good idea.